http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -1507,8 +1507,8 @@ class GKEInterpreter[F[_]](
     // Machine type info
     val maxLimitMemory = machineType.memorySizeInGb
     val maxLimitCpu = machineType.numOfCpus
-    val maxRequestMemory = maxLimitMemory - 5
-    val maxRequestCpu = maxLimitCpu - 3
+    val maxRequestMemory = maxLimitMemory - 15
```

Comment:
Just curious about this - why increase the difference between requests & limits memory/cpu? (15G just seems like a lot.)

Reply:
We added 2 new services to the chart (rabbitmq & tusd) and these were the values I saw on a cluster that had just started Galaxy.